package.json

```diff
@@ -10,7 +10,7 @@
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
     "test:typescript": "tsd",
-    "test:unit": "c8 --100 node --test"
+    "test:unit": "c8 --100 node --test \"**/*.test.js\""
```

Comment:
old version do not support glob pattern.
we need helper library or script for this.

Reply:
yes its annoying.... or we move the types out